src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,95 @@
-export const Pagination = () => {};
+import React from 'react';
+import { getNumbers } from '../../utils';
+import classNames from 'classnames';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (pageNum: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const maxNumPages = Math.ceil(total / perPage);
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const items = getNumbers(
+    (currentPage - 1) * perPage + 1,
+    currentPage * perPage < total ? currentPage * perPage : total,
+  ).map((n: number) => `Item ${n}`);
+  const links = getNumbers(1, maxNumPages).map((n: number) => n);
+  const visibleItems = items.slice(0, perPage);
```

Comment:
The variable `items` appears to be unused in the component rendering. If this is intended for future use, please ignore this comment. Otherwise, consider removing the code to avoid confusion and potential performance issues due to unnecessary computations.